Makes sure that we're not trying to work with a null pointer.

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>

Index: src/tabbar.c
===================================================================
--- mrxvt/src/tabbar.c	(revision 337)
+++ mrxvt/src/tabbar.c	(working copy)
@@ -1327,7 +1327,7 @@
 		if( p.we_wordc > 1)
 		    rxvt_msg( DBG_ERROR, DBG_TABBAR,
 			    "Too many words when expanding %s\n", cwdOption );
-		else
+		else if( NOT_NULL( *p.we_wordv ) )
 		{
 		    filename = *p.we_wordv;
 
